data/mods/gen5/random-teams.ts

```diff
@@ -456,7 +456,7 @@ export class RandomGen5Teams extends RandomGen6Teams {
 				} else if (ability === 'Sturdy') {
 					rejectAbility = (!!counter['recoil'] && !counter['recovery']);
 				} else if (ability === 'Swarm') {
-					rejectAbility = !counter['Bug'];
+					rejectAbility = (!counter['Bug'] || abilities.includes('Hustle'));
```

Comment:
This is the wrong fix because Hustle has a much higher rating. I found the problem.